.ci_support/linux_64_c_compiler_version11cuda_compiler_versionNonecxx_compiler_version11numpy1.21python3.9.____cpython.yaml

```diff
@@ -23,7 +23,7 @@ docker_image:
 glog:
 - '0.6'
 numpy:
-- '1.20'
+- '1.21'
```

Comment:
Is this changed by conda smithy itself?

Reply:
yes, these changes come from conda-smithy